data-import/src/main/resources/folijet/data-import/features/data-import-integration.feature

```diff
@@ -5060,10 +5060,1449 @@ Feature: Data Import integration tests
     And match jobExecution.runBy == '#present'
     And match jobExecution.progress == '#present'
 
-  @Undefined
   Scenario: FAT-944 Match MARC-to-MARC and update Instances, fail to update Holdings and Items
     * print 'Match MARC-to-MARC and update Instance, fail to update Holdings and Items'
 
+    ## Create mapping profile for Instance
+    ## MARC-to-Instance (Marks the Previously held checkbox, changes the statistical code (PTF1), changes status to temporary)
+    Given path 'data-import-profiles/mappingProfiles'
+    And headers headersUser
+    And request
+    """
+    {
+      "profile": {
+        "name": "FAT-944_New: MARC-to-Instance",
+        "incomingRecordType": "MARC_BIBLIOGRAPHIC",
+        "existingRecordType": "INSTANCE",
+        "description": "FAT-944_New: MARC-to-Instance",
+        "mappingDetails": {
+          "name": "instance",
+          "recordType": "INSTANCE",
+          "mappingFields": [
+            {
+              "name": "discoverySuppress",
```

Comment:
A lot of empty elements from mappingFields have no effect when saving the profile. You can delete them.

Reply:
@Abdulkhakimov  - I'll take care about these in future PR. Hope that is fine?